SMLHelper/Assets/ModSprite.cs

```diff
@@ -114,3 +115,100 @@ public ModSprite(TechType type, UnityEngine.Sprite sprite) : this(type, new Atla
         }
     }
 }
+#elif BELOWZERO
+namespace SMLHelper.V2.Assets
+{
+    using System;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// A class that handles a custom sprite and what item it is associated to.
+    /// </summary>
+    internal class ModSprite
```

Comment:
This became two separate classes in one file, and I can see that trying to refactor some of the common elements would just make this messy.

The two versions of `ModSprite` should be separated into two files.

Reply:
Kind of like this? https://github.com/SubnauticaModding/SMLHelper/pull/126/commits/ed5996da0733a71230eeca7d77c0b6cbee2074fe